### ISSUE TITLE
GetFunctionFromIP fails for R2R images inside ModuleLoadFinished

### ISSUE THREAD

**Issue by mjsabby** [dotnet/runtime#6849]
I haven't debugged a whole lot but given that the PerfMap feature takes different paths depending on R2R images or not it's not surprising something isn't working correctly.

That said I'm not sure why EECodeInfo wouldn't have the relevant information for R2R since it clearly can work for NGEN/JIT.

This was discovered when doing dotnet/runtime#6846, except for R2R nothing in the .text section got a successful FunctionID

**Comment by davmason** [comment]
At this point JITCachedFunctionSearchStarted/Finished should work for R2R methods, and the inlining maps are present so EnumNgenModuleMethodsInliningThisMethod will also work for ReJIT with R2R. Closing this, if there is anything outstanding feel free to reopen or create a new issue.
